Logger/BaseLogger.cs

```diff
@@ -2,6 +2,8 @@
 {
     public abstract class BaseLogger
     {
+        public string? Name { get; set; }
```

Comment:
```suggestion
        public string? ClassName { get; set; }
```
For clarity - `Name` is generic.